templates/kdump.conf

```diff
@@ -0,0 +1,3 @@
+path /{{inventory_hostname}}
+core_collector makedumpfile -l --message-level 1 -d 31
+dracut_args --mount "kdump.usersys.redhat.com:/var/www/html/vmcore /mnt/vmcore nfs defaults"
```

Comment:
This looks like site-specific configuration that should be parameterized.

Reply:
Also, line 1 probably should be configurable in case the user has an `inventory_hostname` that isn't as fun for pathnames as well.